pkgs/by-name/il/ilmbase/package.nix

```
@@ -34,7 +34,7 @@ stdenv.mkDerivation {
   '';
 
   meta = with lib; {
-    description = " A library for 2D/3D vectors and matrices and other mathematical objects, functions and data types for computer graphics";
+    description = "A library for 2D/3D vectors and matrices and other mathematical objects, functions and data types for computer graphics";
```

```suggestion
    description = "Library for 2D/3D vectors and matrices and other mathematical objects, functions and data types for computer graphics";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description